.travis.yml

```diff
@@ -9,17 +9,20 @@ dart:
 - dev
 # See https://docs.travis-ci.com/user/languages/dart/ for details.
 dart_task:
-- test: --platform vm
-# Uncomment this line to run tests on the browser.
-# - test: --platform firefox
+  - dartfmt
+  - dartanalyzer
+  - test: --platform vm
+    xvfb: false
+  # Uncomment this line to run tests on the browser.
+  - test: --platform firefox
 
 # Only run one instance of the formatter and the analyzer, rather than running
 # them against each Dart version.
 matrix:
```

Comment:
There is no `-stable`, so no need to exclude them, too. Just delete this section.

Reply:
It is acting as a placeholder for when we resume building across two channels. Do you still want me to remove these lines?